openedx_webhooks/jira/tasks/update_latest_github_activity.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from collections import namedtuple
+import logging
+
+from ...services.jira import jira
+from ...services.jira.utils import make_fields_lookup
+from .utils import convert_to_jira_datetime_string, make_edx_action_choices
+
+logger = logging.getLogger(__name__)
+
+JiraGithubPrFieldNames = namedtuple('_JiraGithubPrFieldNames', (
+    'LAST_UPDATED_AT',
+    'LAST_UPDATED_BY',
+    'LATEST_ACTION',
+    'LATEST_BY_EDX',
+))
+
+JIRA_PR_FIELDS = JiraGithubPrFieldNames(
+    LAST_UPDATED_AT='GitHub PR Last Updated At',
+    LAST_UPDATED_BY='GitHub PR Last Updated By',
+    LATEST_ACTION='GitHub Latest Action',
+    LATEST_BY_EDX='GitHub Latest Action By edX',
+)
+
+JIRA_FIELDS_ID_LOOKUP = make_fields_lookup(JIRA_PR_FIELDS._asdict().values())
+
+JIRA_EDX_ACTION_CHOICES = make_edx_action_choices()
```

Comment:
I would put this function right here, in this file. It's not a general purpose utility, it's only useful to this task.